extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -338,18 +340,28 @@ export class AmpAd3PImpl extends AMP.BaseElement {
 
     const consentPromise = this.getConsentState();
     const consentPolicyId = super.getConsentPolicy();
+    const isConsentV2Experiment = isExperimentOn(this.win, 'amp-consent-v2');
+    const consentStringPromise = (consentPolicyId && isConsentV2Experiment)
+      ? getConsentPolicyInfo(this.getAmpDoc(), consentPolicyId)
+      : Promise.resolve(null);
```

Comment:
nit: Is there a reason why we are resolving null here? 🤔 I think I've been explained why we should do this, but just want to double check.

If no specific reason, we can remove passing null 😄

Reply:
Right. I pass null for other consent-state, consent-shared-data case. So I follow the same pattern and pass a null if the consent string is not defined. There's also the possibility that if `<amp-consent>` doesn't exist on the page, `getConsentPolicyInfo` will resolve with `null` and that's another reason that I want to resolve null here.